tests/ipv6/ipv6_test.go

```diff
@@ -102,6 +105,74 @@ var _ = Describe("{IPv6PxctlFunctional}", func() {
 			})
 			testPxctlCmdForIPv6()
 		})
+
+		// test ip address from pxctl service kvdb endpoints
+		Context("{PxctlServiceKvdbEndpoints}", func() {
+
+			JustBeforeEach(func() {
+				pxctlCmd = ipv6util.PxctlServiceKvdbEndpoints
+				pxctlCmdFull = ipv6util.PxctlServiceKvdbEndpoints
+				expectedIPCount = -1
+				testrailID = 9695435
+			})
+			testPxctlCmdForIPv6()
+		})
+
+		// test ip address from pxctl service kvdb members
+		Context("{PxctlServiceKvdbMembers}", func() {
+
+			JustBeforeEach(func() {
+				pxctlCmd = ipv6util.PxctlServiceKvdbMembers
+				pxctlCmdFull = ipv6util.PxctlServiceKvdbMembers
+				expectedIPCount = -1
+				testrailID = 9695435
+			})
+			testPxctlCmdForIPv6()
+		})
+
+		// test ip address from pxctl service kvdb members
+		Context("{IPv6PxctlAlertsDescription}", func() {
+
+			JustBeforeEach(func() {
+				pxctlCmd = ipv6util.PxctlAlertsShow
+				pxctlCmdFull = ipv6util.PxctlAlertsShow + " -t node -i NodeStateChange -r " + nodes[1].VolDriverNodeID
```

Comment:
nit: change to 
fmt.Sprintf("%s -t node -i NodeStateChange -r %s")

Reply:
I will make this change just wondering why change this to fmt.Sprintf()?

Reply:
just aesthetic, for this example, there's no difference between the two.
only difference is that for using fmt, it's clear we are using a string because of `%s`